packages/validators-util/src/hasMinimumWords/hasMinimumWords.ts

```diff
@@ -0,0 +1,5 @@
+export interface hasMinimumWordsOptions {
```

Comment:
Vi har vel ikke en lintregel på det, men `type` og `interface` pleier typisk starte med stor forbokstav i de kodebasene jeg har jobbet med

```suggestion
export interface HasMinimumWordsOptions {
```

Reply:
Force push uten pull først kanskje? 😄

Reply:
hehe stemmer! Både på stor bokstav og force-push 💯